data/translations/es/state-data.json

```diff
@@ -259,7 +259,12 @@
   "pr": {
     "hp_link_translated": "true",
     "confirm_registration_link_translated": "true",
+    "registration_link_translated": "true",
+    "more_info_link_translated": "true",
     "default_ip_deadline": "hasta 50 días antes del día de las elecciones",
+    "default_online_deadline": "hasta 50 días antes del día de las elecciones",
+    "registration_link_override": "https://ere.ceepur.org/",
+    "confirm_registration_link_override": "https://consulta.ceepur.org/",
     "more_info_link_override": "https://ww2.ceepur.org/es-pr/Paginas/default.aspx"
```

Comment:
These appear to be the same links in the states.json file. Overrides are only necessary when there are alternative links.